client/components/data/query-site-stats/index.jsx

```diff
@@ -38,7 +38,7 @@ class QuerySiteStats extends Component {
 
 		this.props.requestSiteStats( siteId, statType, query );
 		this.clearInterval();
-		if ( heartbeat, isAutoRefreshAllowedForQuery( query ) ) {
+		if ( heartbeat && isAutoRefreshAllowedForQuery( query ) ) {
```

Comment:
another good reason to keep PRs small small small - little things like this slip through easily when we have hundreds of lines to review!